input/cql/UrinarySymptomScoreChangeAfterBenignProstaticHyperplasiaFHIR.cql

```diff
@@ -113,6 +126,9 @@ define "Morbid Obesity Diagnosis or BMI Exam Result Greater Than or Equal to 40
   exists "Morbid Obesity Diagnosis On or Before Follow Up USS Assessment"
     or "BMI Exam Result Greater Than or Equal To 40 During Measurement Period and On or Before Follow Up USS Assessment"
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/ 
```

Comment:
As written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis

Reply:
Updated comment to be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis.